server/routes/api/campaigns.js

```diff
@@ -25,4 +26,16 @@ router.get('/', async (req, res) => {
   }
 })
 
+// Delete endpoint
+router.delete('/:id', async (req, res) => {
+  const id = req.params.id
+  try {
+    await Campaign.query().deleteById(id)
+    res.send({ message: 'Campaign deleted successfully' })
```

Comment:
We should use a pattern of explicitly sending a status code and defining the content-type we are sending back to the client. That will make it easier to understand what is happening in the code and enforce data contracts. You can tweak both `res.send()` calls to:
`res.status(200).json({ message: 'Campaign deleted successfully' }).end()`
`res.status(500).json({ error: error.message }).end()`

Reply:
I corrected the endpoint to send the appropriate status codes and also included the content-type to be sent back to the client